packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useQueueEntries } from '../hooks/useQueueEntries';
+import QueueTableMetricsCard from './queue-table-metrics-card.component';
+import styles from './queue-table-metrics.scss';
+import { type Queue } from '../types';
+
+interface QueueTableMetricsProps {
+  selectedQueue: Queue;
+}
+
+function QueueTableMetrics({ selectedQueue }: QueueTableMetricsProps) {
+  const { t } = useTranslation();
+
+  const allowedStatuses = selectedQueue.allowedStatuses;
+  const { queueEntries } = useQueueEntries({ queue: selectedQueue.uuid, isEnded: false });
+
+  return (
+    <div className={styles.metricsBorder}>
+      <QueueTableMetricsCard value={queueEntries.length} headerLabel={t('totalPatients', 'Total Patients')} />
```

Comment:
use `totalCount` returned from `useQueueEntries()` instead. `queueEntries.length` is technically incorrect because the REST endpoint limits the max returned result count to 50 by default.

This also means the per-status metrics are not calculated correctly as is, but I don't really know what's the best solution to that. We have a [related ticket](https://openmrs.atlassian.net/browse/O3-3015) regarding this.

Reply:
Thanks @chibongho ! I have updated the PR to use totalCount. Thanks!

Reply:
If the same `totalCount` can't be applied to the per-status counts @cioan , then one way to solve this would be via multiple calls to the `queue-entry-metrics` endpoint that is here:  https://github.com/openmrs/openmrs-module-queue/blob/main/omod/src/main/java/org/openmrs/module/queue/web/QueueEntryMetricRestController.java .  Not sure if this will be too much overhead with the multiple calls, but it would provide the most accurate data.

Reply:
Looking pretty good @cioan .  A few additional comments to address